apps/epl/src/epl_viz_map.erl

```diff
@@ -144,6 +146,16 @@ push_focused_connection(Source, Target, RegionName, {N, W, D}, A, Vizceral) ->
     NewR = push_connection(Source, Target, {N,W,D}, A, Region),
     push_region(RegionName, NewR, NewV).
 
+%% @doc Clears all focused nodes from `RegionName` node.
+-spec clear_focused_nodes_inside_region(RegionName :: name(), Viz :: map()) ->
+  map().
+clear_focused_nodes_inside_region(RegionName, Viz) ->
```

Comment:
What do you think about using `clear_focused_nodes_and_conns_inside_region(...)` here?